### PR TITLE
new icon: perl (original, plain)

### LINKED ISSUE #185: Icon Request: Perl
* Perl
* perl
* Raku

### PR DESCRIPTION
**Pull request by mattkoskela**
Closes #185

### CODE REVIEW

**Comment by mattkoskela** [comment]
#185

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please fix the issues mentioned [here](https://github.com/devicons/devicon/runs/2413616985). If it asks you to change the PR title, you must change the commit title instead. To do so, follow the steps [here](https://stackoverflow.com/questions/9103396/how-do-i-edit-a-previous-git-commit). In order to change the commit title, use `r` instead of `s`.

**Comment by Thomas-Boi** [comment]
@Panquesito7 It seems like the issue is the "can't make connection to OS" again.

File: devicon.json [hunk 1]
```diff
@@ -2484,6 +2484,20 @@
...
+            "programming",
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ]
+        },
+        "color": "#212177"
```
Comment (Thomas-Boi) [inline devicon.json:2499]: Add this attribute:
```
,
"aliases": []
```
While we don't enforce the aliases, right now the script still look for an aliases attribute.

**Review by Thomas-Boi — CHANGES_REQUESTED** [review]

File: devicon.json (hunk 1 above)
Comment (Panquesito7) [reply devicon.json:2499]:
```suggestion
        "color": "#212177",
        "aliases": []
```

File: devicon.json [hunk 2]
```diff
@@ -2484,6 +2484,21 @@
...
+        "name": "perl",
+        "tags": [
+            "programming",
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ]
```
Comment (Thomas-Boi) [inline devicon.json:2497]: You'll need to add this attribute in the `versions` as well:
```
"font": [
  "plain"
]
```

**Review by Thomas-Boi — CHANGES_REQUESTED** [review]

File: devicon.json (hunk 2 above)
Comment (Thomas-Boi) [reply devicon.json:2497]: I would recommend checking the neighbouring entry to ensure all the attributes are correct. You can also check the [`CONTRIBUTING.md`](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#updateDevicon) for more details

Comment (Panquesito7) [inline devicon.json:2497]:
```suggestion
            ],
            "font": [
                "plain",
            ]
```

**Review by Panquesito7 — DISMISSED** [review]

File: devicon.json [hunk 3]
```diff
@@ -2484,6 +2484,24 @@
...
+            "programming",
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain",
```
Comment (Panquesito7) [inline devicon.json:2499]:
```suggestion
                "plain"
```

**Review by Thomas-Boi — CHANGES_REQUESTED** [review]
The file names in the folder are in the wrong format. Please append "perl-" to the front of each file. 

See [this](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#svg-standards)

File: icons/perl/perl-original.svg [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Panquesito7) [inline icons/perl/perl-original.svg:1]: Maybe you should optimize the SVG with the [SVG Viewer](https://www.svgviewer.dev/) (keep the `viewBox` and add a line at the end). Same with the `plain` version.

Comment (Thomas-Boi) [reply icons/perl/perl-original.svg:1]: Panquesito7 is right. Once this is optimized, I'll merge your icon @mattkoshela

**Comment by mattkoskela** [comment]
Cleaned up the svg files and ran the svg viewer optimization!

**Review by Thomas-Boi — APPROVED** [review]
Sweet, everything looks good @mattkoskela! Thanks for the icon, Perl is one that a couple people has asked us about.

**Comment by mattkoskela** [comment]
Do you guys have a rough idea when the next release will go out?

**Comment by Thomas-Boi** [comment]
I think it will be at least for the next 2-3 weeks. It depends on how many PRs we get. 

We will also be going over some of the stale PRs in the next couple days. Once we get those in (about 16 icons), I think that'll be enough for another release.

**Pull request merged**